client/src/auth/sign-up.tsx

```diff
@@ -38,27 +38,11 @@ export default function SignUpApp() {
     );
   }
 
-  if (searchParams.get("errors")) {
-    console.warn("Errors", searchParams.get("errors"));
-    const errors = JSON.parse(searchParams.get("errors") || "{}");
-    return (
-      <div>
-        <h2>Sign-in errors</h2>
-        <p>An error occurred trying to complete sign in.</p>
-        <pre>{JSON.stringify(errors, null, 2)}</pre>
-        <p>
-          We apologize for the inconvenience,{" "}
-          <Link to={`/${locale}/signin`}>please try signing-in again</Link>.
-        </p>
-      </div>
-    );
-  }
-
   const csrfMiddlewareToken = searchParams.get("csrfmiddlewaretoken");
   const provider = searchParams.get("provider");
   if (!csrfMiddlewareToken || !provider) {
     return (
-      <div>
+      <div className="notecard error">
```

Comment:
Is it abuse to use `notecard` for these things?

Reply:
Nope, all good for now. We could do with something dedicated to these instances, but for now, this works fine.